aot/src/test/resources/com/dylibso/chicory/approvals/ApprovalTest.verifyBrTable.approved.txt

```diff
@@ -55,33 +55,24 @@ public final class com/dylibso/chicory/$gen/CompiledMachine implements com/dylib
 
   public static func_0(ILcom/dylibso/chicory/runtime/Memory;Lcom/dylibso/chicory/runtime/Instance;)I
     ILOAD 0
-    INVOKESTATIC com/dylibso/chicory/$gen/CompiledMachine$AotMethods.checkInterruption ()V
```

Comment:
This is an interesting change, do you think it's not needed?

Reply:
Yes, it's not needed because none of the switch targets are a backward jump.

Reply:
The reason why I'm a bit unsure here is that we are not running the `InterruptionTest` on the aot compiled code.
But I'd prefer to keep the ball rolling other than fussing over a detail that can be reverted at any time.